vita3k/kernel/src/thread/sync_primitives.cpp

```diff
@@ -728,7 +746,11 @@ int eventflag_set(KernelState &kernel, const char *export_name, SceUID thread_id
     }
 
     const std::lock_guard<std::mutex> event_lock(event->mutex);
-    event->flags |= flags;
+    if (do_clear) {
```

Comment:
oh, also, you can remove { } , for this if and else, no need it if have only one value on return :)